### ISSUE TITLE
Multiple IP Adress In One Server

### ISSUE THREAD

**Issue by ahmetugur91** [Jigsaw-Code/outline-sdk#341]
Hi, i have one ubuntu server, lets say i have 2 IP adress, is it possible to serving 2 IP adress as VPN same time? If its yes how? Thank you.

**Comment by zaferatli** [comment]
yes its possible, tunnel works with ports and its connect with ip address with properly configured network its could works. probably no one can answer how question because its depend on network architecture -- its could be change even same hosting firms.
